hack/generate-buildah-remote.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
 SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-podman run -v "$SCRIPTDIR"/..:/data quay.io/redhat-user-workloads/rhtap-build-tenant/multi-arch-controller/multi-arch-controller:taskgen-221979523603b730feca730aeb9d43f24a3c1e67  --buildah-task=/data/task/buildah/0.1/buildah.yaml --remote-task=/data/task/buildah-remote/0.1/buildah-remote.yaml
+podman run -v "$SCRIPTDIR"/..:/data quay.io/redhat-user-workloads/rhtap-build-tenant/multi-arch-controller/multi-arch-controller:taskgen-e008b351d34597c08b6731f21de37b3103662431  --buildah-task=/data/task/buildah/0.1/buildah.yaml --remote-task=/data/task/buildah-remote/0.1/buildah-remote.yaml
```

Comment:
This update (and the corresponding buildah-remote update) doesn't seem connected to the rest of this change. Could you split it into a separate commit?

Reply:
without this the buildah-remote task and this one would end up out of sync with the images they are using.

Reply:
It doesn't seem like this script touches rpm-ostree though :thinking:

Reply:
Yea, there is another PR. It has been a long day.

Reply:
Ah, ok. We can probably drop the change in the buildah-remote task as well then :slightly_smiling_face: